test-apps/basic/src/index.tsx

```diff
@@ -14,6 +14,16 @@ let test_function = (word) => {
 }
 
 const App = () => {
+
+  let testObject = {
+    someFunction() {
+      return api.asApp().requestConfluence(route`/rest/api/3/test`);
```

Comment:
This test case is actually not being caught. We're getting baited by the `foo()` function call that also makes an `asApp()` call, which causes the `run` entrypoint to trigger the authorization vulnerability. If you remove the call to `asApp()` in `foo()`, then you'll notice that we aren't reporting any vulnerabilities here, even though we should.

You can either remove the call to `foo()` in the `App` function or you can create a separate module for testing object method calls.

For actually solving the problem of us not tracking method invocations, you'll probably have to look into the interpreter/checkers to see how we resolve calls. My gut tells me that we currently treat method calls as a no-op.

Reply:
Thank you very much for checking. This has been resolved. I think I introduced a bug when resolving merge conflicts 😅